src/css/main.less

```diff
@@ -361,6 +361,8 @@ fieldset[disabled] .btn-info.active {
   padding-bottom: 12px;
   padding-top: 11px;
   padding-left: 0;
+  text-align: center;
+  width: 187px; /* sidebar 202px - .container-fluid 15px padding-left */
```

Comment:
Yeah we should introduce variables at some point. Also, we should probably split the file into components...